### PR TITLE
Hero header

### PR DESCRIPTION
**Pull request by humeniukd**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,7 +124,24 @@ <h3 class="shopping-cart__title-header">Recently added items</h3>
 
   </div>
   <!-- / .header__cart -->
-
+  <!-- .header__hero -->
+    <div class="header__hero">
+      <div class="hero-content">
+        <p class="title__hero">Take charge of your life<br>start your own business</p>
```
Comment (jacekpolak) [inline index.html]: Please use h3 or h4 as a title

**Comment by jacekpolak** [comment]
https://snag.gy/tYlyh9.jpg Please change line-height to 1 and top space above title to 15px to make sure it will looks good with this image on iPhone portrait instead of current layout https://snag.gy/X6WVJI.jpg

**Comment by jacekpolak** [comment]
https://snag.gy/ubLqG4.jpg on retina display during scaling curve sometimes showing 1px image. Please position it at bottom: -1px to prevent it.

File: index.html [hunk 2]
```diff
@@ -124,7 +124,24 @@ <h3 class="shopping-cart__title-header">Recently added items</h3>
...
+        <p class="hero__description">Start selling great products and<br>become an Amway Business Owner</p>
+        <a href="#" class="btn btn-hero__play">
+          <svg class="svg-icon hero-play">
+            <use xlink:href="#icon-play-circle"></use>
+          </svg>
+          <span>Watch video</span>
+        </a>
+      </div>
+      <img class="hero-image" src="./media/hero_BusinessOp.jpg">
```
Comment (jacekpolak) [inline index.html]: please add alt for image

File: src/scss/header/hero.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.hero-image {
+  width: 100%;
+  height: auto;
+}
+
+.hero-curve {
+  width: 100%;
+  height: 24px;
+  fill: #fff;
```
Comment (jacekpolak) [inline src/scss/header/hero.scss]: please map local colors from variables

File: src/scss/header/hero.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.hero-content {
+  text-align: center;
+  position: absolute;
+  top: 30px;
+  margin-top: 30px;
+  width: 100%;
+  @include font-size(15px);
+  color: $alabaster;
```
Comment (yura-matsko) [inline src/scss/header/hero.scss]: Use local variables

File: src/scss/app.scss [hunk 5]
```diff
@@ -12,3 +12,4 @@
 @import "base/forms";
 
 @import "components/header/cart";
+@import "header/hero";
```
Comment (jacekpolak) [inline src/scss/app.scss]: lets add it int component header folder.

**Pull request merged**